po/pt.po

```diff
@@ -8,26 +8,34 @@ msgid ""
 msgstr ""
 "Project-Id-Version: extension-manager\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-10-28 14:17+0200\n"
-"PO-Revision-Date: 2023-01-03 11:02+0000\n"
+"POT-Creation-Date: 2023-11-22 00:48+0200\n"
+"PO-Revision-Date: 2024-02-15 17:25+0000\n"
 "Last-Translator: Hugo Carvalho <hugokarvalho@hotmail.com>\n"
 "Language-Team: \n"
 "Language: pt\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: Poedit 3.2.2\n"
+"X-Generator: Poedit 3.4.2\n"
 
 #: data/com.mattjakeman.ExtensionManager.desktop.in.in:3
 #: data/com.mattjakeman.ExtensionManager.metainfo.xml.in.in:6
 msgid "@app_title@"
 msgstr "@app_title@"
 
+#: data/com.mattjakeman.ExtensionManager.desktop.in.in:11
+msgid "Manage GNOME Shell Extensions"
+msgstr "Gerir as extensões do GNOME Shell"
+
+#. Translators: Do not translate
+#: data/com.mattjakeman.ExtensionManager.desktop.in.in:12
+msgid "extension;manager;"
+msgstr "extensão;gestor;"
```

Comment:
These are search terms, they are mostly made to search for the application by its English name even though the application is translated, since the name of the application is already translated.

```suggestion
#. Translators: Do not translate
#: data/com.mattjakeman.ExtensionManager.desktop.in.in:12
msgid "extension;manager;"
msgstr "extension;manager;"
```

or

```suggestion
#. Translators: Do not translate
#: data/com.mattjakeman.ExtensionManager.desktop.in.in:12
msgid "extension;manager;"
msgstr "extension;manager;extensão;gestor;"
```

Reply:
Done. I've updated the file and the corresponding PR